mne/bem.py

```diff
@@ -332,3 +337,108 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
```

Comment:
I don't think that the function in `mne/bem.py` should look for the `SUBJECT` env var, none of our other functions do this. As such, it's a little bit dangerous to do it actually. I'd prefer the command line tool not look for it, either, but it's understandable to have it look for the env var because that duplicates the functionality of the original shell script. Make sense?